src/azure-cli/azure/cli/command_modules/acr/connected_acr.py

```diff
@@ -0,0 +1,192 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from msrest.exceptions import ValidationError
+from knack.log import get_logger
+from knack.util import CLIError
+from azure.cli.core.commands.client_factory import get_subscription_id
+from ._utils import (
+    get_registry_by_name,
+    validate_managed_registry,
+    user_confirmation
+)
+
+DEFAULT_MODE = 'registry'
+DEFAULT_LOG_LEVEL = 'Info'
+DEFAULT_MESSAGE_TTL = 'P2D'
+
+logger = get_logger(__name__)
+
+
+def acr_connected_acr_create(cmd,
+                             client,
+                             registry_name,
+                             connected_acr_name,
+                             repositories,
+                             resource_group_name=None,
+                             mode=DEFAULT_MODE,
+                             parent=None,
+                             sync_schedule=None,
+                             sync_window=None,
+                             auto_update_enabled=True,
+                             log_level=DEFAULT_LOG_LEVEL,
+                             sync_message_ttl=DEFAULT_MESSAGE_TTL,
+                             sync_audit_logs_enabled=True):
+
+    registry, resource_group_name = get_registry_by_name(
+        cmd.cli_ctx, registry_name, resource_group_name)
+    subscription_id = get_subscription_id(cmd.cli_ctx)
+
+    SyncProperties, ParentProperties = cmd.get_models(
+            'SyncProperties', 'ParentProperties')
+    sync_properties = SyncProperties(
+        TokenId=None, #Create Token? The resource ID of the ACR token used to authenticate the connected registry to its parent during sync. 
+        Schedule=sync_schedule,
+        MessageTtl=sync_message_ttl,
+        SyncWindow=sync_window,
```

Comment:
`syncWindow` is required server side. We support a minimum sync window of `PT1H`